src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -88,13 +77,13 @@ void func_808FD080(s32 idx, ColliderJntSph* collider, Vec3f* arg2) {
         collider->elements[idx].dim.modelSphere.radius * collider->elements[idx].dim.scale;
 }
 
-void BossGanon2_SetObjectSegment(BossGanon2* this, GlobalContext* globalCtx, s32 objectId, u8 arg3) {
```

Comment:
Not sure how I feel about `setRSPSegment` but I don't really have any suggestions.

Reply:
rsp segment is far too general, object is fine imo cause its for segment 6 specifically

Reply:
nvm, i misunderstood and thought thats what the function was renamed to

Reply:
yeah what dragorn has is fine, its a var for setting the segment on the rsp vs just setting it on the cpu. there isnt really a higher level way to explain it that is more elegant

Reply:
I'm basing this name on http://n64devkit.square7.ch/pro-man/pro11/11-01.htm#02

> The RSP maintains an associative table of up to 16 segment ID's and their base addresses.